exampleCourse/questions/demo/addHTMLtable/question.html

```diff
@@ -0,0 +1,36 @@
+
+
+<pl-question-panel>
+
+  <div class="card my-2">
+      <div class="card-header">
+          Example Table 1:
+      </div>
+      <div class="card-body">
+          {{{params.mytable}}}
+      </div>
+ </div>
+
+ <div class="card my-2">
+     <div class="card-header">
+         Example Table 2: adding table properties
+     </div>
+     <div class="card-body">
+         <table style="width:30%" border="black">
+           {{{params.mytable2}}}
+         </table>
+     </div>
+</div>
+
+<div class="card my-2">
+    <div class="card-header">
+        Example Table 3: get table from console output
+    </div>
+    <div class="card-body">
+<pre>
+{{params.mytable3}}
+</pre>
```

Comment:
```suggestion
<pre>{{params.mytable3}}</pre>
```

This will remove the extra blank line at the end. It also means you can intent the whole thing, I think.

Reply:
I will try that.